apps/x509.c

```diff
@@ -138,6 +138,7 @@ const OPTIONS x509_options[] = {
      "Number of days until newly generated certificate expires - default 30"},
     {"preserve_dates", OPT_PRESERVE_DATES, '-',
      "Preserve existing validity dates"},
+    {"issu", OPT_ISSU, 's', "Set or override certificate issuer"},
```

Comment:
I'd propose to rename the option to `set_issuer`

Reply:
Should we alias `-set_subject` to `-subj` for consistency?

Reply:
Yes, that would make sense.